### DISCUSSION TITLE
How to change local cache to remote cache

### DISCUSSION THREAD

**Discussion by tdoor** [keycloak/keycloak#14627] [Q&A]
As our service running , we have two keycloak HA cluster .
When change realms or users in keycloak1 and save , then refresh keycloak2 , the setting can not sync .
We prepare  to change the local cache to remote cache in realms/users/authorization/keys , can you help me support some demo?Thanks.

here are standalone-ha.xml , we want to change local-cache to remote-cache on infinispan

![image](https://user-images.githubusercontent.com/54176978/192727234-07a707c2-b1cf-45e8-977f-a774ae8dc7c4.png)

**Comment by sschu** [comment]
The changes should sync, as there are different distributed caches used for syncing the local caches. If that does not work for you, there is probably a problem with your cluster setup.

**Reply by tdoor** [reply]
This change can be sync , but when we remove   <expiration lifespan="1000"/>   ,  the cluster can not sync the data Immediatly.
The 1000ms lifespan will lower our machine. thank you for your answer.
